### PR TITLE
N18: fishgame

### PR DESCRIPTION
**Pull request by not-fl3**
Part of #455

### CODE REVIEW

File: content/posts/newsletter-018/index.md [hunk 1]
```diff
@@ -105,6 +105,25 @@ The project is still at a “very” early stage of development (Dec. 2020).
...
+_Fish game gameplay footage._
+
+[Fishgame][fishgame] is an online multiplayer game, created in a
+collaboration between [Nakama][nakama], an open-source scalable game
+server, and the [Macroquad](https://github.com/not-fl3/macroquad/) game engine.
+
+The game is going to showcase nakama multiplayer capabilities for the rust
+language world.
+
+This month the game got a public, multiplayer HTML5 build. [Play it online!][fishgame-itch
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-018/index.md:120]:
```suggestion
This month the game got a public, multiplayer HTML5 build. [Play it online!][fishgame-itch]
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Looks good, just one minor issue with a link.

Also, could you replace the GIF with this optimized version? Should cut the size down by about a megabyte :)

fishgame

**Comment by not-fl3** [comment]
> Also, could you replace the GIF with this optimized version? Should cut the size down by about a megabyte :)

Sure, updated!
But this is awesome, I tried really hard to make it fit 2.5Mb, and actually there were more optimizations available! 
How you made it? :)

**Comment by 17cupsofcoffee** [comment]
> How you made it? :)

@ozkriff linked me to https://ezgif.com/optimize and it seems to work really well :)

**Review by 17cupsofcoffee — APPROVED** [review]

**Comment by 17cupsofcoffee** [comment]
Thank you for the PR :)

**Pull request merged**